src/Domain/Configuration.php

```diff
@@ -179,6 +226,7 @@ private function resolveConfig(array $config): void
             'directory' => (string) getcwd(),
             'exclude' => [],
             'add' => [],
+            'requirements' => [],
```

Comment:
Is it possible to add some check on what is passed into $config['requirements'] ? 

Whats happen if I put in my `phpinsights.php` something like this: 

```php
return [
    // ...
    'requirements' => [
         'unexpected_key' => 50,
    ]
]
```

I think we can handle it with an other OptionResolver

Reply:
Wouldn't it just ignore that? As it is not a key we use?